atlas/AtlasMl/atlasml/ml/similarity_measures.py

```
@@ -0,0 +1,111 @@
+import numpy as np
+from scipy.spatial.distance import jaccard, euclidean, cosine
+from typing import Union, List
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)